### PR TITLE
Fixed NativeWindow.GetText() missing last character

### PR DESCRIPTION
**Pull request by Jijjy**

### CODE REVIEW

File: WinApi/Windows/NativeWindow.cs [hunk 1]
```diff
@@ -43,7 +43,7 @@ public string GetText()
             if (size > 0)
             {
                 var sb = new StringBuilder(size);
```
Comment (prasannavl) [inline WinApi/Windows/NativeWindow.cs:45]: This also has to be size + 1. 
Simply declaring a new variable `var length = size +1` and using them later would be ideal.

**Comment by prasannavl** [comment]
@Jijjy Nice catch! Thanks. All set.

File: WinApi/Windows/NativeWindow.cs (hunk 1 above)
Comment (Jijjy) [reply WinApi/Windows/NativeWindow.cs:45]: @prasannavl Are you sure? size seems to equal the length of text returned.

Comment (prasannavl) [reply WinApi/Windows/NativeWindow.cs:45]: Ah. You're right. My bad, replied being half sleepy. `nMaxCount` is just the number of characters. For some reason, had it mind that it was the buffer size.

**Pull request merged**